trade-finance-manager-api/src/v1/controllers/auth/auth-provider-config.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+require('dotenv').config();
+
+const {
+  AZURE_SSO_AUTHORITY,
```

Comment:
We could add some parsing of this to validate

Reply:
I'm going to leave this for now as it'll require introducing a new pattern here, and reading these env vars are self contained inside this config file.